packages/playground/remote/src/lib/boot-playground-remote.ts

```diff
@@ -232,6 +239,9 @@ export async function bootPlaygroundRemote() {
 		throw e;
 	}
 
+	wpFrame.addEventListener('load', () => {
+		webApi.downloadWordPressAssets();
```

Comment:
Since we are using PHP to unzip now, you could do the fetch here and then run the unzip helper function using webApi as the PHP client to avoid reimplementing the extraction logic in PHP.

Reply:
Uint8array is a transferrable object so that would just work

Reply:
On the other hand, it's nice to have the entire bundle loading logic contained in the worker. Alright, let's stick to a method call. However, let's rename it to something more specific, like `backfillStaticFilesRemovedFromMinifiedBuild` (or something shorter if possible)